example-get-started-experiments/code/src/train.py

```diff
@@ -68,6 +68,7 @@ def train():
         models_dir = Path("models")
         models_dir.mkdir(exist_ok=True)
         learn.export(fname=(models_dir / "model.pkl").absolute())
+        torch.save(learn.model, (models_dir / "model.pth").absolute())
         live.log_artifact(
```

Comment:
@daavoo If we logged the tar file of the model, then we wouldn't need to specify the file name `model.tar.gz` in the deployment script, right? The deployment script would be able to get the file name from the Git tag itself, which means that a single deployment script could be used for deploying several models with different names. Right?